framework/src/dirackernels/VectorPostprocessorPointSource.C

```diff
@@ -0,0 +1,83 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "VectorPostprocessorPointSource.h"
+#include "MooseUtils.h"
+#include "DelimitedFileReader.h"
+
+registerMooseObject("MooseApp", VectorPostprocessorPointSource);
+
+defineLegacyParams(VectorPostprocessorPointSource);
```

Comment:
Not needed for new objects anymore, just FYI

Reply:
I tried not using it and I get: 
dirackernels/vectorPostprocessor_point_source.2dTransient: dyld: Symbol not found: __Z11validParamsI30VectorPostprocessorPointSourceE15InputParametersv
dirackernels/vectorPostprocessor_point_source.2dTransient:   Referenced from: /Users/mundlb/projects/moose/framework/libmoose-opt.0.dylib
dirackernels/vectorPostprocessor_point_source.2dTransient:   Expected in: flat namespace